### PR TITLE
community/index.md: add info about matrix room

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

@Luap99 @TomSweeneyRedHat @rhatdan PTAL

### CODE REVIEW

**Review by Luap99 — COMMENTED** [review]
LGTM

File: community/index.md [hunk 1]
```diff
@@ -9,9 +9,10 @@ title: Join Our Community
 
 We want your feedback, issues, patches, and involvement in the development of Podman. See Contributing for how to submit issues & PRs, and chat with us on Slack, IRC, or Email.
 
-## Slack and IRC
+## Slack, IRC and Matrix
+
+The Podman developers often hang out on the #CRI-O channel in [Kubernetes Slack](https://slack.k8s.io/). Note that the devs are generally around during CEST and Eastern Time business hours, so be patient if you're in another time zone. There's also Podman users and developers on IRC on the `#podman` channel on `libera.chat` as well as `[#podman:matrix.org](https://matrix.to/#/#podman:matrix.org)` on `matrix`. The `matrix` room and `libera.chat` channel have been bridged so joining either one should be sufficient.
```
Comment (TomSweeneyRedHat) [inline community/index.md:14]: LGTM
@lsm5 do you know if we could also tie discord in with the matrix?

Comment (lsm5) [reply community/index.md:14]: do we have a discord already?

Comment (lsm5) [reply community/index.md:14]: I don't see a built-in bridge, but there's 3rd party support. https://matrix.org/docs/projects/bridge/matrix-appservice-discord

**Comment by lsm5** [comment]
@TomSweeneyRedHat i see the embedded matrix URL looks ugly. Is it because of the `#` in the beginning? https://podman.io/community/

**Comment by TomSweeneyRedHat** [comment]
I think you want to take the backticks off the front and back.

[#podman:matrix.org](https://matrix.to/#/#podman:matrix.org)

**Pull request merged**